page.py

```diff
@@ -207,18 +207,18 @@ def _play_target_sound(self):
         if self._activity.mode in ['letter', 'find by letter']:
             aplay.play(os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][1])[:-1])
+                    self._media_data[self.target][1]).replace('\n', ' ').replace('\r', ''))
         elif self._activity.mode == 'picture':
             aplay.play(os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][1])[:-1])
+                    self._media_data[self.target][1]).replace('\n', ' ').replace('\r', ''))
             GLib.timeout_add(1000, aplay.play, os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][0])[:-1])
+                    self._media_data[self.target][0]).replace('\n', ' ').replace('\r', ''))
         else:
             aplay.play(os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][0])[:-1])
+                    self._media_data[self.target][0]).replace('\n', ' ').replace('\r', ''))
```

Comment:
I suppose it would be better if we use `str.strip` to remove non-printable characters like `\n\r`.

Reply:
Please have a look now

Reply:
Cool, its better now!